int8 led_id
std_msgs/ColorRGBA color

---
int32 status
string message